.github/workflows/CI.yml

```diff
@@ -515,6 +518,8 @@ jobs:
             public.ecr.aws/${{ env.ECR_REPO }}:latest
           cache-from: type=registry
           cache-to: type=inline
+          platforms : linux/amd64, linux/arm64
+          build-args: ${{ needs.e2etest-preparation.outputs.version }}
```

Comment:
Mind if I ask what we need this flag thought?

Reply:
From the slack chat earlier. It is a way adding a label called version with the version of the collector.

Reply:
docker inspect {image}
under labels it will give the adot version. This is good for people that use latest tag instead of a version for their tag.

Reply:
Reasonable.